conf/ocp_version/ocp-4.5-config.yaml

```diff
@@ -3,7 +3,7 @@
 RUN:
   client_version: '4.5.0-0.nightly'
 DEPLOYMENT:
-  installer_version: "4.5.0-0.nightly-2020-06-19-043844"
+  installer_version: "4.5.0-rc.2-x86_64"
```

Comment:
FYI: you can also use --ocp-version to override this.

Reply:
I think that `--ocp-version` will work with `4.5` or `4.6` and not with the exact build

Reply:
you are right, it just picks the config file based on version match.